llvm/test/Transforms/EntryExitInstrumenter/pre-inliner-instrumentation.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -passes="default<O1>" -S < %s | FileCheck %s
+; RUN: opt -passes="thinlto-pre-link<O2>" -S < %s | FileCheck %s
+; RUN: opt -passes="thinlto-pre-link<O2>,thinlto<O3>" -S < %s | FileCheck %s
```

Comment:
no need to run `thinlto<O3>` here, but we should test that `thinlto`/`lto` don't run this pass by adding different RUN lines with a different `FileCheck %s --checkprefix=NO`

Reply:
Ah, good idea! I still went with O2 for lto/thinlto because even at O1 it does not inline the alwaysinline leaf, oops.